packages/commonwealth/client/scripts/views/components/UpvotePopover/UpvotePopover.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import app from 'state';
+import CWPopover, {
+  UsePopoverProps,
+} from '../component_kit/new_designs/CWPopover/CWPopover';
+import { User } from '../user/user';
+import './UpvotePopover.scss';
+
+type UpvotePopoverProps = {
+  upvoters: string[];
+} & UsePopoverProps;
+
+export const UpvotePopover = ({
+  upvoters,
+  ...popoverProps
+}: UpvotePopoverProps) => {
+  const maxVisibleUpvotingAccounts = 3;
+
+  const createUpvoterList = (upvoterAddresses) => {
+    const slicedUpvoters = upvoterAddresses
+      .slice(0, maxVisibleUpvotingAccounts)
+      .map((upvoterAddress) => {
+        return (
+          <div key={upvoterAddress} className="upvoter-row">
+            <User
+              userAddress={upvoterAddress}
+              // set to 1inch for use on components page
+              userCommunityId={app.chain?.id || '1inch'}
+              shouldLinkProfile
+            />
+            <div className="vote-weight">5x</div>
+          </div>
+        );
+      });
+
+    if (upvoterAddresses.length > maxVisibleUpvotingAccounts) {
+      slicedUpvoters.push(
+        <div
```

Comment:
let's use <CWText> with type `caption` to align styling with figma

Reply:
Done!